geometry_msgs/PoseStamped hand
geometry_msgs/PoseStamped culbuto_1

# Hand: 3 valeurs (X, Y, Z) x 10 points temporels = 30D
# Culbuto: 3 valeurs (X, Y, Z) x 10 points temporels = 30D
